src/utils.php

```diff
@@ -154,3 +155,25 @@ function validateEditCarDataFields(string $name, int $year_made = 0, float $zero
     }
     return $err_msg;
 }
+
+function filterBrand(string $brand): string | false
+{
+    $model = new CarModel(make_db());
```

Comment:
Rather than instantiating the model in here, you should instead instantiate it outside the function, and pass it in as a parameter